code/modules/mob/living/simple_animal/hostile/ipc_zombie.dm

```diff
@@ -23,7 +23,6 @@
 	max_n2 = 0
 	minbodytemp = 0
 	speed = 4
-	see_in_dark = 8
```

Comment:
why?

Reply:
Because we don't use this var, it's set only once and at max
![image](https://github.com/Aurorastation/Aurora.3/assets/65877598/9c085fcc-ba54-4390-8a1c-2bdcc4274be7)